### DISCUSSION TITLE
Default Orgs and Projects

### DISCUSSION THREAD

**Discussion by rdmorris** [digitalocean/supabase-on-do#19] [Ideas]
I was thinking it would make sense to update the /terraform/files/.env.tftpl file from
`
############
# Studio - Configuration for the Dashboard
############

STUDIO_DEFAULT_ORGANIZATION=Default Organization
STUDIO_DEFAULT_PROJECT=Default Project

`

to 

`
############
# Studio - Configuration for the Dashboard
############

STUDIO_DEFAULT_ORGANIZATION=${TF_DEFAULT_ORGANIZATION}
STUDIO_DEFAULT_PROJECT=${TF_DEFAULT_PROJECT}
`

and add the following to /terraform/terraform.tfvars.example

`
############
# Studio - Configuration for the Dashboard
############

STUDIO_DEFAULT_ORGANIZATION  = "Default Organization"
STUDIO_DEFAULT_PROJECT             ="Default Project"
`

**Comment by timothymamo** [comment]
@rdmorris

This has been added with the following [PR](https://github.com/digitalocean/supabase-on-do/pull/21).

You will need to set the variables `studio_org` and `studio_project` in your `tfvars` file otherwise they will be given the default values of [Default Organization](https://github.com/digitalocean/supabase-on-do/blob/main/terraform/variables.tf#L162) and [Default Project](https://github.com/digitalocean/supabase-on-do/blob/main/terraform/variables.tf#L168).
